### ISSUE TITLE
Rumbledrug Buff

### ISSUE THREAD

**Issue by SuckMySocks** [HaloSpaceStation/HaloSpaceStation13#1444]
As of current rumbledrugs are completely useless for what their intended purpose is suppose to be. They give little-to-no buffs and only serve to kill the player. Gonna leave it to Pantas to explain his findings after his tests with rumble.

**Comment by daPotato** [comment]
+1 - Choked down 31 rumbles on Koenigsegg's test server and had barely any benefit.

**Comment by PantasTheDank** [comment]
Alright, so far my findings were that even with 6 pills consumed,I didn't get much from it,except imminent death.
First off I took a rumble drugs pill,grinded it,and slowly injected all of it,nothing. I didn't even start getting my organgs hurt.
Then I started eating all the pills,and my character started shaking,I expected something to happen,I raced with another guy and we had the same speed although he didn't consume the drugs,we both had no gear,and yet we were matched in speed.
I then fired a sound rifle on me and I got downed,I expected the drugs would provide the afromentioned Pain Immunity or atleast Ressistance effect,which I didn't get even with rumble in my system.
After eating all the pills eventually I started to puke and eventualy my vision was whiter and I got knocked down and started dying,my liver went down probably.
In conclusion I only gained slow and painful death from the pills.

**Comment by BDpuffy420** [comment]
concentrated hyperzine was added, but I'm not sure its in the current rumbledrug mix
